src/EventSubscriber/CreateNewWishlistSubscriber.php

```diff
@@ -81,25 +56,30 @@ public function onKernelRequest(RequestEvent $event): void
             return;
         }
 
+        $request = $this->requestStack->getMainRequest();
+        $currentPath = $request->getPathInfo();
+        if (!str_starts_with($currentPath, '/wishlist')) {
```

Comment:
Wouldn't it be better to store '/wishlist' as a const?

Reply:
yup

Reply:
Alse, shouldn’t we use routing name instead of path? I believe it would be easier to maintain this part of code.

Reply:
I think the URI is good enough here :vulcan_salute: